css/style.css

```diff
@@ -0,0 +1,596 @@
+  transform: rotate(-45deg) translate(-6px, 2px);
+}
+
+.hamburger.is-active::after {
+  transform: rotate(45deg) translate(-10px, -6px);
+}
+
+.hamburger.is-active .bar {
+  opacity: 0;
+}
+
+.mobile-nav {
+  position: fixed;
+  display: block;
+  top: 0;
+  left: 100%;
+  min-height: 100vh;
+  z-index: 98;
+  background-color: #d3d3d3;
+  margin-top: 60px;
+  height: 3px;
+}
+
+ul {
+  text-decoration: none;
+  list-style: none;
+  text-align: center;
+}
+
+.mobile-nav ul {
+  display: block;
+  width: 100%;
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: 600;
+  transition: 0.4s;
+  margin: 0;
+  margin-top: 4rem;
+  margin-left: -1rem;
+}
+
+.mobile-nav ul li {
+  padding: 10px 0;
+}
+
+.mobile-nav ul li a {
+  text-decoration: none;
+  color: #272a31;
+  font-family: "lato", sans-serif;
+}
+
+.footer-flex .footer-logo a {
+  text-decoration: none;
+  color: #272a31;
+}
+
+.mobile-nav ul li a:hover {
+  color: #ec5242;
+}
+
+.mobile-nav.is-active {
+  left: 0;
+  width: 100%;
+  margin-top: 0;
+}
+
+@media (min-width: 768px) {
+  .mobile-nav {
+    display: none;
+  }
+
+  .hamburger {
+    display: none;
+  }
+}
+
+@font-face {
+  font-family: Cocogoose;
+  src: url(../fonts/Cocogoose\ Pro-trial.ttf);
+}
+
+@media only screen and (min-width: 768px) {
+  .top-bar-section {
+    display: block;
+    position: relative;
+    background-color: black;
+    padding-right: 10.5rem;
+  }
+
+  .top-bar-container {
+    width: 85%;
+    margin: auto;
+  }
+
+  .top-bar {
+    display: flex;
+    list-style: none;
+    justify-content: flex-end;
+    align-items: center;
+    margin: 0;
+    padding-left: 0;
+  }
+
+  .top-bar li {
+    background-color: black;
+    color: white;
+    padding-inline: 0.8%;
+    cursor: pointer;
+  }
+
+  .top-bar-section .top-bar-container .top-bar li h4 {
+    font-size: 15px;
+    color: white;
+    font-weight: 400;
+    padding-top: 4px;
+  }
+
+  .top-bar-section .top-bar-container .top-bar li h4:hover {
+    color: #ec5242;
+  }
+
+  .navbar-expand-md {
+    padding-left: 16rem;
+  }
+
+  .navbar .container-fluid .btn-outline-success {
+    margin-right: 16rem;
+    width: 130px;
+  }
+
+  .navbar .container-fluid .navbar-brand img {
+    width: 20%;
+  }
+
+  .navbar-light .navbar-nav .nav-link:hover {
+    color: #ec5242 !important;
+  }
+
+  .first-section {
+    margin-top: 0 !important;
+    width: 100%;
+    padding-left: 16rem;
+  }
+
+  .first-section .first-section-container {
+    margin-top: 6rem;
+  }
+
+  .first-section .first-section-container h1 {
+    font-size: 47px !important;
+    width: 51%;
+  }
+
+  .first-section .first-section-container .first-section-paragraph .first-paragraph {
+    font-size: 13px !important;
+    font-weight: 600;
+    width: 48% !important;
+    margin-top: 0 !important;
+    border: 7px solid white;
+    color: #272a31;
+    background: #f7f7f7;
+  }
+
+  .first-section .first-section-container .second-heading {
+    padding-top: 7rem !important;
+  }
```

Comment:
Please try to avoid using !important. 

Please view this [article](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) on the Mozilla Dev network docs about specificity and why !important is bad because it makes debugging difficult